website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -69,6 +69,8 @@ Development or Deployment credentials in a dbt project. Simply enter your Snowfl
 username (specifically, the `login_name`) and the corresponding user's Snowflake `password`
 to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowflake user.
 
+**Tip**: The schema field in the 'Developer Credentials' section is a required field.
```

Comment:
nitpick with the word `tip`. doesn't that mean something that helps someone work more efficiently? in this case it doesn't strike me as a tip but a requirement...

Reply:
Good flag, thank you @dataders ! i'll change it to note